app/code/core/Mage/Core/Model/App/Emulation.php

```diff
@@ -106,12 +108,12 @@ public function startEnvironmentEmulation($storeId, $area = Mage_Core_Model_App_
      */
     public function stopEnvironmentEmulation(Varien_Object $initialEnvironmentInfo)
     {
-        $this->_restoreInitialInlineTranslation($initialEnvironmentInfo->getInitialTranslateInline());
-        $initialDesign = $initialEnvironmentInfo->getInitialDesign();
+        $this->_restoreInitialInlineTranslation($initialEnvironmentInfo->getData('initial_translate_in_line'));
```

Comment:
As far as I read this line it should be

`$this->_restoreInitialInlineTranslation($initialEnvironmentInfo->getData('initial_translate_inline'));`

Reply:
Will be reverted.